kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml

```diff
@@ -85,13 +85,14 @@
    </p>
             <div xml:id="kzz7yh-f51825-Dd1e146">
                <head xml:id="kzz7yh-f51825-Hd1e148">Quaestio 1</head>
+               <head type="question-title">Utrum aliquid de natura celesti fiat de essentia ipsius ade</head>
                <p xml:id="kzz7yh-f51825-d1e150">
       Questio. I 
       <lb ed="#V" n="33"/>PRimo ostendo quod aliquid de 
       na<lb ed="#V" n="34" break="no"/>tura celesti fiat 
-      <lb ed="#V" n="35"/>e essentia ipsius ade philosophus. 16 de animalibus. 
-      di<lb ed="#V" n="36" break="no"/>cit quod in semine triplex est calor scilicet celestis igneus et 
-      <lb ed="#V" n="37"/>animalis. Sed accidens non est stne subiecto: ergo 
+      <lb ed="#V" n="35"/>de essentia ipsius ade <ref>philosophus. 16 de animalibus.</ref> 
+      di<lb ed="#V" n="36" break="no"/>cit quod in semine<!--s or f? see line below too--> triplex est calor scilicet celestis igneus et 
+      <lb ed="#V" n="37"/>animalis. Sed accidens non est stne<!--word--> subiecto: ergo 
```

Comment:
The word should be "sine" but the type print is a little wonky, so I am unsure of whether I should make the definitive correction.

Reply:
@jeffreycwitt

Reply:
sine is ok here
```suggestion
      <lb ed="#V" n="37"/>animalis. Sed accidens non est sine subiecto: ergo 
```